File: Python/20August/prog3.py
dc = {"Germany":"Berlin", "Italy":"Venice", "France":"Versailles", "Canada":"Quebec City"}
cnt = 0
for k,v in dc.items():
	if cnt % 2 == 0:
		print(k)
		cnt += 1
	else:
		print(v)
		cnt += 1

File: Java/DailyFlash/17Sept/prog3.java
class Prog3 {

	public static void main(String[] args) {
		
		for (int i = 1; i <= 5; i++) {
			int cnt = 0;
			for (int j = 1; j <= 5; j++) {	
				if(i >= 2 && i < 5){
					if(j >= 3)
						System.out.print(" ");
					else
						System.out.print(i + " ");
				}
				else
					System.out.print(i + " ");
			}
			System.out.println();
		}
	}
}